packages/core/src/core.ts

```diff
@@ -191,8 +198,11 @@ export async function group<T>(name: string, fn: () => Promise<T>): Promise<T> {
  * @param     name     name of the state to store
  * @param     value    value to store
  */
-export function saveState(name: string, value: string): void {
-  issueCommand('save-state', {name}, value)
+export function saveState(
+  name: string,
+  value: string | number | boolean
```

Comment:
if we accept any, shouldnt we fix the type to be any? and comment should describe behavior (e.g. tojson for objects)